lib/generators/doorkeeper/templates/enable_pkce_migration.rb

```diff
@@ -0,0 +1,6 @@
+class EnablePkce < ActiveRecord::Migration
+  def change
+    add_column :oauth_access_grants, :code_challenge, :string, null: true
+    add_column :oauth_access_grants, :transformation_method, :string, null: true
```

Comment:
I would suggest renaming this from `transformation_method ` to `code_challenge_method` to keep it consistent.